HRTool/src/main/java/app/controllers/ReportRestController.java

```diff
@@ -0,0 +1,212 @@
+	
+	@Autowired
+	ProjectRepository projectRepository;
+	
+	@Autowired
+	ProjectInfoRepository projectInfoRepository;
+
+	@Autowired
+    private ServletContext servletContext;
+	
+	@RequestMapping("/cv")
+	/**
+	 * Sends generates cv file to response.
+	 * @param id employee's id
+	 * @return File as an array of bytes.
+	 * @throws IOException
+	 */
+	public ResponseEntity<byte[]> generateRtf(@RequestParam("id") int id) throws IOException {
+		Employee e = employeeRepository.findOne(id);
+		List<TagCloud> allEducation = tagCloudRepository.findByTipTagCloud(TagCloudEnum.Education);
+		Set<TagCloud> education = new HashSet<>();
+		for (TagCloud tc : allEducation) {
+			if (tc.getEmployees().contains(e)) {
+				education.add(tc);
+			}
+		}
+		List<TagCloud> allLanguage = tagCloudRepository.findByTipTagCloud(TagCloudEnum.ForeignLanguage);
+		Set<TagCloud> language = new HashSet<>();
+		for (TagCloud tc : allLanguage) {
+			if (tc.getEmployees().contains(e)) {
+				language.add(tc);
+			}
+		}
+		Set<ProjectInfo> pinfos = e.getProjectInfos();
+		Set<TagCloud> databases = new HashSet<>();
+		Set<TagCloud> ides = new HashSet<>();
+		Set<TagCloud> technologies = new HashSet<>();
+		Map<Project, List<TagCloud>> projects = new HashMap<>();
+		for(ProjectInfo pi : pinfos) {
+			List<TagCloud> tempTech = new ArrayList<>(); 
+			for(TagCloud tc : pi.getTagClouds()) {
+				if(tc.getTipTagCloud().equals(TagCloudEnum.Technologie)) {
+					tempTech.add(tc);
+					technologies.add(tc);
+				}
+				else if(tc.getTipTagCloud().equals(TagCloudEnum.Database)) {
+					databases.add(tc);
+				}
+				else if(tc.getTipTagCloud().equals(TagCloudEnum.IDE)) {
+					ides.add(tc);
+				}
+			}
+			projects.put(pi.getProject(),tempTech);
+			
+		}
+		File file = null;
+		try {
+			file = CVGenerator.generate(servletContext,e,education,language,projects, technologies, databases, ides);
+		} catch (FileNotFoundException e2) {
+			e2.printStackTrace();
+		} catch (DocumentException e2) {
+			e2.printStackTrace();
+		}
+		String employeeName = e.getNameEmployee().replace(" ", "_");
+		String fileName = employeeName + "_CV.rtf";	
+		Path path = Paths.get(servletContext.getRealPath("/temp/")+ fileName);
+    	byte[] data = null;
+		try {
+			data = Files.readAllBytes(path);
+		} catch (IOException e1) {
+			e1.printStackTrace();
+		}
+    	HttpHeaders headers = new HttpHeaders();
+    	headers.setContentType(MediaType.parseMediaType("application/rtf"));
+    	headers.add("content-disposition", "inline; filename=" + fileName);
+    	if(file != null) {
+    		file.delete();
+    	}
+    	ResponseEntity<byte[]> response = new ResponseEntity<byte[]>(data, headers, HttpStatus.OK);
+        return response;
+	}
+	
+	@RequestMapping("/seniority")
+	/**
+	 * Sends seniority pie chart as an array of bytes.
+	 * @return Seniority pie chart
+	 */
+	public ResponseEntity<byte[]> generateChart() {
+		try {
+			ChartGenerator.generatePieChart(employeeRepository.findAll());

```

Comment:
Nije dobro to sto se rezultat  ne vrati nego se implicitno ocekuje da ce snimiti na putanju ./seniority-chart.pdf
Vajdica je bar staviti i neki datum u ime, da se spreci gazenje starih.